packages/auth/package.json

```diff
@@ -27,6 +27,7 @@
   "dependencies": {},
   "devDependencies": {
     "@standardnotes/config": "^2.0.0",
+    "@standardnotes/snjs": "2.7.20",
```

Comment:
hmm wondering if that won't be circular - snjs will depend on auth and auth will depend on snjs. 

Also seems an overkill for auth to require the whole SNJS. 

We have to figure out this dependecies mashup I think - wdyt?

Reply:
Yup true story! Maybe we could have a subset of ContentType and ComponentArea specified on auth? Con for that would be we'd have to make changes on both packages when we add new options

Reply:
hmm `auth` doesn't seem to be a good place for `ContentType` and `ComponentArea`. 

I'd suggest extract those into a new package, and make both `auth` and `snjs` dependant on that new package.

Lacking a good name for the package - @mobitar any ideas :) ?

Reply:
maybe `content` or `item` ? Trying to come up with something more general that could umbrella at least a couple of things

Reply:
Perhaps we could move all these types to a separate features package?

Reply:
good idea! do you need help in creating the package :) ?

Reply:
Created the new package, left ContentType and ComponentArea and made the new features package dependent on both auth and SNJS (but neither of them depends on features). Let me know if it looks alright 🙂